hydrogym/core.py

```diff
@@ -196,6 +196,15 @@ def dot(self, q1, q2):
         u2, _ = q2.split()
         return fd.assemble(inner(u1, u2) * dx)
 
+    def get_inertia(self):
+        pass
+
+    def get_damping(self):
+        pass
+
+    def get_state(self):
```

Comment:
I think `state` might be a confusing name here, since `q = [u, p]^T` is what people will usually think of as the "state" of the flow.  Maybe `ctrl_state` or `actuator_state`?